src/vcpkg/commands.cpp

```diff
@@ -150,75 +150,114 @@ namespace vcpkg
         format_command_usage_entry(table, CommandPackageInfoMetadata);
         format_command_usage_entry(table, CommandPortsdiffMetadata);
+        format_command_usage_entry(table, CommandSearchMetadata);
         format_command_usage_entry(table, CommandUpdateMetadata);
         table.blank();
 
-        table.header(msg::format(msgPackageManipulationCommmands));
+        table.header(msg::format(msgPackageManipulationHeader));
         format_command_usage_entry(table, CommandAddMetadata);
         format_command_usage_entry(table, CommandAddVersionMetadata);
-        format_command_usage_entry(table, CommandCiCleanMetadata);
-        format_command_usage_entry(table, CommandCiMetadata);
-        format_command_usage_entry(table, CommandCiVerifyVersionsMetadata);
         format_command_usage_entry(table, CommandCreateMetadata);
         format_command_usage_entry(table, CommandEditMetadata);
         format_command_usage_entry(table, CommandEnvMetadata);
         format_command_usage_entry(table, CommandFormatManifestMetadata);
         format_command_usage_entry(table, CommandHashMetadata);
         format_command_usage_entry(table, CommandInitRegistryMetadata);
         format_command_usage_entry(table, CommandNewMetadata);
-        format_command_usage_entry(table, CommandRegenerateMetadata);
+        format_command_usage_entry(table, CommandUpdateBaselineMetadata);
         table.blank();
 
-        table.header(msg::format(msgOtherCommands));
+        table.header(msg::format(msgOtherCommandsHeader));
+        format_command_usage_entry(table, CommandCiMetadata);
+        format_command_usage_entry(table, CommandCiVerifyVersionsMetadata);
         format_command_usage_entry(table, CommandContactMetadata);
-        format_command_usage_entry(table, CommandDownloadMetadata);
         format_command_usage_entry(table, CommandFetchMetadata);
-        format_command_usage_entry(table, CommandVersionMetadata);
-        format_command_usage_entry(table, CommandVsInstancesMetadata);
+        format_command_usage_entry(table, CommandIntegrateMetadata);
         table.blank();
 
         table.header(msg::format(msgForMoreHelp));
         table.format("vcpkg help topics", msg::format(msgHelpTopicsCommand));
         table.format(msg::format(msgCmdHelpTopic), msg::format(msgHelpTopicCommand));
-        table.blank();
-        table.format(msg::format(msgResponseFileCode), msg::format(msgHelpResponseFileCommand));
+        table.format("vcpkg help commands", msg::format(msgCmdHelpCommandsSynopsis));
+        table.format(msg::format(msgCmdHelpCommands), msg::format(msgCmdHelpCommandSynopsis));
         table.blank();
         table.example(msg::format(msgHelpExampleCommand));
 
         msg::println(LocalizedString::from_raw(table.m_str));
     }
+
+    void print_full_command_list()
+    {
+        HelpTableFormatter table;
+        format_command_usage_entry(table, CommandAcquireMetadata);
```

Comment:
This seems easy to miss when adding new commands.

Do we have a list of all commands somewhere else we could loop through, skipping based on undocumented, etc?